net-print/kyocera-1x2x-mfp-driver/kyocera-1x2x-mfp-driver-1.1203-r1.ebuild

```diff
@@ -0,0 +1,131 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Printer descriptions (PPDs) and filters for Kyocera 1x2x MFP"
+HOMEPAGE="http://www.kyoceramita.it/"
+SRC_URI="LinuxDrv_${PV}_FS-1x2xMFP.zip"
+
+LICENSE="kyocera-mita-ppds"
+SLOT="0"
+
+# TODO: Someone please test the ebuild on ~x86.
+KEYWORDS="-* ~amd64 ~x86"
```

Comment:
It'd be better if you didn't add `~x86` keyword then, and we can open keywordreq if someone really needs it. However, 32-bit x86 systems are pretty scarce these days, so it's probably not worth the effort.